content/doc/developer/publishing/releasing-experimental-updates.adoc

```diff
@@ -2,9 +2,25 @@
 title: Publishing Experimental Plugin Releases
 layout: developer
 ---
+The link:https://github.com/jenkinsci/plugin-installation-manager-tool[Plugin Installation Manager tool] with link:/doc/developer/plugin-development/incrementals/[incremental] builds is often a preferred technique :
 
-To simplify delivery of beta versions of plugins to interested users, the Jenkins project publishes an _experimental update center_.
-It includes alpha and beta versions of plugins, which aren't usually included in the regular update center.
+- Plugin installation manager tool enables easier import into an instance of Jenkins by downloading plugins and their respective dependencies into a folder.
+
+- It allows the administrator to use incremental builds, without requiring a "beta" release.
+
+- And provides handy information about the plugins being downloaded (such as, available updates or security warnings and many more).
+
+- Incremental builds, on the other hand, provides the user with the flexibility to create incremental versions of jenkins core or an upstream plugin with dependencies, while providing the ability to switch between and keep track of incremental versions.
+
+- It makes it easier to test and explore unreleased plugin versions.
+
+The other way, is to simplify the delivery of plugin beta versions to interested users. 
+The Jenkins project publishes an _experimental update center_.
+The experimental update center includes alpha and beta versions of plugins, which aren't usually included in the regular update center. 
+
+NOTE: The experimental update center is deprecated to use in favor of incrementals, as described above.
+
+The following will give detailed explanation of this method:
```

Comment:
```suggestion
```
I would remove this sentence, it doesn't help/provide anything useful to the user